modules/ballerina-http/src/main/ballerina/ballerina/net/http/annotation.bal

```diff
@@ -23,7 +23,7 @@ public annotation configuration attach service<> {
     webSocket webSocket;
 }
 
-public annotation webSocket {
+public annotation webSocket attach service {
```

Comment:
Should this be service<ws> instead of just service   ?

Reply:
Done. It should be attached to HTTP service.